core/common/src/main/java/alluxio/Constants.java

```diff
@@ -169,5 +169,36 @@
   // Ufs fingerprint
   public static final String INVALID_UFS_FINGERPRINT = "";
 
+  // Metrics-related constants
+  public static final String DIRECTORIES_CREATED_METRICS_NAME = "DirectoriesCreated";
```

Comment:
Do you think it makes sense to move the metrics names constants to a `MetricNames` class? Then we could remove the `METRICS_NAME` part, e.g. `MetricNames.FILE_BLOCK_INFOS_GOT`. Also good not to overwhelm this class if we want to add a lot more metrics.

Reply:
I take a look at the metric name management. Most of the metrics names (except those DefaultFileSystemMetrics I used) were put in the alluxio-core-common alluxio.metrics.ClientMetrics/WorkerMetrics classes.
These classes seem to be only responsible for storing metric names.

I am thinking if I should create a MasterMetrics class and store the defaultFileSystemMaster metrics inside this class.

If we create a MetricNames class, that will also work. Then it may be more reasonable to put all the metrics names inside it(including those defined in ClientMetrics and WorkerMetrics class). I guess they create separate classes for maintaining the metrics names to let users be more aware of the metrics type(from master, from client or from worker).

Which way do you think is better? @aaudiber  Thanks!

Reply:
I think both approaches are good, leave it up to you

Reply:
Put file system master metrics names into MasterMetrics.java, PTAL, thanks